src/app/containers/ContactsContainer.js

```diff
@@ -163,12 +169,16 @@ const ContactsContainer = ({ location, history }) => {
     const handleCheckAll = (checked = false) => handleCheck(contacts.map(({ ID }) => ID), checked);
     const handleUncheckAll = () => handleCheckAll(false);
 
-    const handleMerge = (beMergedContacts) => {
+    const handleMerge = () => {
         const currentContactID = getCurrentContactID();
         createModal(
-            <MergeModal contacts={beMergedContacts} contactID={currentContactID} userKeysList={userKeysList} />
+            <MergeModal contacts={mergeableContacts} contactID={currentContactID} userKeysList={userKeysList} />
         );
     };
+    const handleImport = () => createModal(<ImportModal userKeysList={userKeysList} />);
+    const handleExport = (contactGroupID) =>
+        createModal(<ExportModal contactGroupID={contactGroupID} userKeysList={userKeysList} />);
+    const handleGroups = () => history.push({ ...location, pathname: `/contacts/settings` });
```

Comment:
Do you really need to pass `location` to settings state? `history.push('/contacts/settings')` is not enough?
If you really need it, please change:
```suggestion
    const handleGroups = () => history.push({ ...location, pathname: '/contacts/settings' });
```

Reply:
It's actually necessary to clear the history in this case, so it should be `history.push('/contacts/settings')`!